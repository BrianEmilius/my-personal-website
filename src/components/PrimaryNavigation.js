import React from "react";
import AniLink from "gatsby-plugin-transition-link/AniLink";
import styles from "./PrimaryNavigation.module.css";

export default function PrimaryNavigation() {
  return (
    <nav>
      <ul className={styles.primaryNavigation}>
        {/* <li>
          <Link to="/work" className={styles.primaryNavigation__link}>
            Work
          </Link>
        </li>
        <li>
          <Link to="/contact" className={styles.primaryNavigation__link}>
            Contact
          </Link>
        </li> */}
        <li>
          <AniLink fade to="/about" className={styles.primaryNavigation__link}>
            About
          </AniLink>
        </li>
        <li>
          <AniLink fade to="/blog" className={styles.primaryNavigation__link}>
            Blog
          </AniLink>
        </li>
      </ul>
    </nav>
  );
}
