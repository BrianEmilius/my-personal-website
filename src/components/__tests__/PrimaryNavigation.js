import React from "react";
import renderer from "react-test-renderer";

import PrimaryNavigation from "../PrimaryNavigation";

describe("PrimaryNavigation", function() {
  it("renders correctly", function() {
    const tree = renderer.create(<PrimaryNavigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
