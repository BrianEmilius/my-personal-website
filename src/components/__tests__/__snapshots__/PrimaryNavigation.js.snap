// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryNavigation renders correctly 1`] = `
<nav>
  <ul
    className="primaryNavigation"
  >
    <li>
      <a
        className="primaryNavigation__link"
        href="/blog"
      >
        Blog
      </a>
    </li>
  </ul>
</nav>
`;
